### PR TITLE
Documentation for Sign In with OIDC

### PR DESCRIPTION
**Pull request by J0**
## What kind of change does this PR introduce?

This PR adds supporting documentation for:

https://github.com/supabase/gotrue/pull/360/files

as well as

https://github.com/supabase/gotrue-js/pull/207

Please don't merge until the above two PRs go in!

## What is the current behavior?

NA

## What is the new behavior?

NA

## Additional context

Add any other context or screenshots.

### CODE REVIEW

**Review by awalias — APPROVED** [review]

File: web/spec/supabase.yml [hunk 1]
```diff
@@ -576,6 +576,22 @@ pages:
...
           });
           ```
+      - name: Sign in with Open ID Connect.
+        description: |
+          Supabase support signing in with Open ID Connect.
+        js: |
+          ```js
+           const { user, session, error } = await supabase.auth.signIn({
+            phone: '+13334445555',
+            password: 'some-password',
```
Comment (awalias) [inline web/spec/supabase.yml:586]: you can remove password and phone fields for this snippet

**Review by awalias — CHANGES_REQUESTED** [review]

File: web/spec/supabase.yml (hunk 1 above)
Comment (awalias) [reply web/spec/supabase.yml:586]: I think?

Comment (J0) [reply web/spec/supabase.yml:586]: yep, thanks for the catch!

**Review by CabanillasJ2467 — APPROVED** [review]

**Review by kangmingtay — CHANGES_REQUESTED** [review]
can we also add the other alternative which accepts the issuer as a param? 

```
curl -X "POST" "https://PROJECT_REF.supabase.co/auth/v1/token?grant_type=id_token" \
     -H 'apikey: apikey' \
     -H 'Content-Type: application/json; charset=utf-8' \
     -d '{
              "client_id": "client_id",
              "nonce": "nonce",
              "id_token": "id_token",
              "issuer": "issuer"
      }'
```

**Comment by osdiab** [comment]
Checking if this is going to be merged/if OIDC is ready for production use. Would love to try it in my project. Thanks!

**Comment by J0** [comment]
Hey! 

Unfortunately, we're not quite ready to release  OIDC yet and OIDC isn't quite ready for production use. We're going to close for now but will revisit in the near future.

**Pull request closed without merging**